applications/HelloIOIO/src/androidTest/java/ioio/examples/hello/HelloIOIOSmokeTest.kt

```diff
@@ -27,6 +27,6 @@ class HelloIOIOSmokeTest {
     @Test
     fun smokeTestSimplyStart() {
         onView(withId(R.id.button)).check(matches(isDisplayed()))
-        Screenshot.takeScreenshot(BuildConfig.FLAVOR + "-smoke")
+        Screenshot.takeScreenshot(BuildConfig.APPLICATION_ID + "-smoke")
```

Comment:
I prefer `Screenshot.takeScreenshot("smoke")` otherwise the name will be very long.